docs/pages/inboxes/create-a-client.mdx

```diff
@@ -52,10 +52,21 @@ The `dbEncryptionKey` client option is not used by the Browser SDK for due to te
 
 To learn more about database operations, see the [XMTP MLS protocol spec](https://github.com/xmtp/libxmtp/blob/main/xmtp_mls/README.md).
 
+### Offline mode
+
+When you pass `allowOffline = true` to [`Client.create()`](#create-a-client), or build a client with [`Client.build()`](#build-an-existing-client) and a locally stored `inboxId`, the client operates in offline mode as follows:
+
+- Skips all network requests (identity ledger, key packages, etc.)
+- Works entirely from the local database
+- Postpones identity registration if no inbox ID exists
+- Can be synchronized later with `syncAllConversations()` or by recreating the client without the offline flag.
+
 ## Create a client
 
 To call `Client.create()`, you must pass in a required `signer` and can also pass in any of the optional parameters covered in [Configure an XMTP client](#configure-an-xmtp-client).
 
+You can pass in `allowOffline = true` to create a client in [offline mode](#offline-mode).
```

Comment:
You can not create a client offline only build

Reply:
The client has to already be created and stored in the local database for offline mode to work